cli/xsum_os_specific.c

```diff
@@ -135,6 +140,13 @@ static int XSUM_stat(const char* infilename, XSUM_stat_t* statbuf)
 #ifndef XSUM_NO_MAIN
 int main(int argc, const char* argv[])
 {
+#ifdef __OpenBSD__
```

Comment:
Please explain why `pledge()` is desirable and why it's only for OpenBSD.

Reply:
A comment now explains the promises, but I've omitted further reasoning how pledge(2) is an OpenBSD-only syscall.